cocotb/binary.py

```diff
@@ -101,16 +100,20 @@ def __init__(self, value=None, n_bits=None, bigEndian=True,
                  binaryRepresentation=BinaryRepresentation.UNSIGNED,
                  bits=None):
         """
-        Kwagrs:
-            value (string or int or long): value to assign to the bus
-
-            n_bits (int): Number of bits to use for the underlying binary
-                          representation
-
-            bigEndian (bool): Interpret the binary as big-endian when
-                                converting to/from a string buffer.
-
-            bits (int): Deprecated: Compatibility wrapper for n_bits
+        Args:
+            value (str or int or long, optional): 
```

Comment:
In the google style docstrings it is acceptable to have the description of the arguments on the same line as their name/type, just for future reference.  Not sure if there is an opinion on this within the community at large.  This is just a comment, no need to change!